senders/client_factory.go

```diff
@@ -185,14 +185,25 @@ func TracesPort(port int) Option {
 
 // SDKMetricsTags sets internal SDK metrics.
 func SDKMetricsTags(tags map[string]string) Option {
+	// prevent caller from accidentally mutating this option.
+	tags = copyTags(tags)
```

Comment:
Can we rename this variable if the same name `tags` is confusing? May be `copiedTags` or something else?